homeassistant/components/airnow/sensor.py

```diff
@@ -158,6 +168,11 @@ def native_value(self) -> StateType:
         """Return the state."""
         return self.entity_description.value_fn(self.coordinator.data)
 
+    @property
+    def available(self) -> bool:
+        """Return if sensor data exists."""
+        return self.entity_description.value_fn(self.coordinator.data) is not None
+
```

Comment:
I'm not sure if this is needed.
Is there a benefit to showing "Unavailable" instead of "Unknown"?

Reply:
I think it's just semantics for me. If someone wants to filter and not display entities that are unavailable, it makes sense to me to mark things as available or unavailable. Unknown almost seems like a temporary lack of data. But I don't know if there's a home-assistant approved way of thinking about this.

Reply:
https://developers.home-assistant.io/docs/core/entity#generic-properties
`available`: Indicate if Home Assistant is able to read the state and control the underlying device.

Reply:
Okay, reverted that commit.